src/core/tests/ov_tensor_test.cpp

```diff
@@ -212,6 +297,29 @@ TEST_F(OVTensorTest, cannotCreateTensorWithWrongStrides) {
     }
 }
 
+TEST_F(OVTensorTest, cannotCreateStringTensorWithWrongStrides) {
+    ov::Shape shape = {2, 3};
+    std::string data[] = {"abdcd efg hi", "01234", "xyz  ", "   ", "$%&%&& (*&&", "", "\n ", "\t "};
```

Comment:
what if I want to store UNICODE symbols?

Reply:
No problem with it using std::string

Reply:
should not we use `std::wstring` to it?

Reply:
I would not consider `wstring` for reasons: 
1. this is most of Windows specific type. And Linux does not need.
2. wstring changes bitstream representing Unicode stream if you write it into char[] array. In the meantime, string does not and stores original bitstream.

So let us stay on `string` representation for now.